modules/standard/Types.chpl

```diff
@@ -209,31 +209,31 @@ proc chpl__maxIntTypeSameSign(type t) type {
 // due to lack of consensus for the name.
 //
 
-pragma "no doc"
+/* Returns `true` if the argument is a `bool` value. */
 proc isBoolValue(e)      param  return isBoolType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a `int` value. */
 proc isIntValue(e)       param  return isIntType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a `uint` value. */
 proc isUintValue(e)      param  return isUintType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a `real` value. */
 proc isRealValue(e)      param  return isRealType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a `imag` value. */
 proc isImagValue(e)      param  return isImagType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a `complex` value. */
 proc isComplexValue(e)   param  return isComplexType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a `string` value. */
 proc isStringValue(e)    param  return isStringType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a `bytes` value. */
 proc isBytesValue(e)     param  return isBytesType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a value of one the following types: 'int', 'uint'. */
```

Comment:
Can you change your single quotes `'` into backticks `` ` ``, and make these lines fit into 80 characters?

Reply:
Change all single quotes right?

Reply:
Yes, please. Types should always be like `` `int` ``, and not  `'int'`

Reply:
I have made the changes.

Reply:
Why did we separate them with slashes now? Why not `` `int`, `uint` ... `` as it used to be in your initial PR?

Reply:
then it will be more than 80 chars

Reply:
You can add a new line :)

Reply:
ohh, mb.😅

Reply:
Btw while checking after making docs the line is properly aligned (previous one `int`, `uint` ...). should I still fit in 80 chars or just make it as the initial PR.

Reply:
The rendered documentation doesn't need to fit in any number of columns. It is just the source code.

I am not sure if I answered the question you're asking, though.